### ISSUE TITLE
Will not pass "The wind blows intensely."

### ISSUE THREAD

**Issue by Socialblurr** [Admiral-Billy/Pokerogue-App#39]
(on android) I've beaten this eternius 4 different ways, and no matter how I complete the battle it gets stuck at "The wind blows intensely"![Screenshot_20240514_152546_PokeRogue Offline.jpg](https://github.com/Admiral-Billy/Pokerogue-App/assets/136138889/1ba686c1-48cc-4237-b007-29205644184f)

**Comment by Socialblurr** [comment]
I have beaten classic once already. Ive got 4 set move combos to beat this with my current team and all get stuck on that without completing the run. And I've tried leaving it for 5 minutes to see if it would load past that too and that didn't work either.

**Comment by Dracaerys2a** [comment]
Same for me on windows. I play in offline only. I tried to export my data, delete the app, install the latest version (14/05/2024), still the same issue with that run. 
![image](https://github.com/Admiral-Billy/Pokerogue-App/assets/95142314/c18669f8-47ac-47a4-ac12-2ccb109fed74)
I checked the console included in the app, seems like the problem is the connexion refused error to localhost:8001:
![image](https://github.com/Admiral-Billy/Pokerogue-App/assets/95142314/26d67ce4-5a20-4bd1-9cb2-0965185c915a)
 Someone knows a fix ?
